### PR TITLE
✨ [Amp story panning media] Support portrait images

### LINKED ISSUE #31515: [Panning media] Handle base zoom of aspect ratio wider than image
If viewport aspect ratio is wider than the image, base minimum zoom on height being 100% instead of width.

cc @ampproject/wg-stories

### PR DESCRIPTION
**Pull request by processprocess**
Adds support for portrait images by checking if the aspect ratio of the container is less than the image.
Updates example template.

[Demo](https://discovery-ec239.web.app/examples/amp-story/amp-story-panning-media.html#page=painting-portrait-0)
Fixes #31515

### CODE REVIEW

File: extensions/amp-story-panning-media/0.1/amp-story-panning-media.js [hunk 1]
```diff
@@ -201,8 +197,14 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
         '"lock-bounds" requires "width" and "height" to be set on the amp-img child.'
       );
     }
-    // TODO(#31515): When aspect ratio is portrait, containerWidth will be used for this.
-    const percentScaledToFitViewport = containerHeight / ampImgHeight;
+
+    const containerRatio = containerWidth / containerHeight;
```
Comment (gmajoulet) [inline extensions/amp-story-panning-media/0.1/amp-story-panning-media.js:201]: Is this code assuming that `amp-story-panning-media` takes up the full page? I'm wondering about the cases where `panning-media`'s height would be, say, 50%, or when a `amp-story-grid-layer` has an `aspect-ratio`

Comment (processprocess) [reply extensions/amp-story-panning-media/0.1/amp-story-panning-media.js]: Nice! It's updated to measure the element instead of the page.

**Review by gmajoulet — APPROVED** [review]

**Review by mszylkowski — APPROVED** [review]
Looks good.

Just a note that the image looks like the image follows the `cover` CSS logic for background images (in terms of `cover` vs `contain`), so we could explain that in the documentation. A visualization would help as well understand this behavior.

**Comment by processprocess** [comment]
> Looks good.
> 
> Just a note that the image looks like the image follows the `cover` CSS logic for background images (in terms of `cover` vs `contain`), so we could explain that in the documentation. A visualization would help as well understand this behavior.

Good call. I'll update the docs with behavior and a diagram.

**Pull request merged**